src/erp12/cbgp_lite/task.clj

```diff
@@ -16,7 +16,10 @@
 
 (defn vars-for-types
   [types]
-  (set (keys (lib/lib-for-types types))))
+  (let [type-thing 
+        (set (keys (lib/lib-for-types types)))]
+    (println "Vars for this problem:" type-thing)
+    type-thing))
```

Comment:
I like the idea of logging all the vars used during a run!

I think we should use the logging library rather than a print statement so that we can control if it goes to stdout or a file and we get additional information in the log (timestamp, location in the code, etc). It should be as easy as using `log/info` instead of `println`.

Also, this log statement would probably be more appropriate at the location we start a `run`. Somewhere around [here](https://github.com/erp12/cbgp-lite/blob/master/benchmarks/erp12/cbgp_lite/benchmark/ga.clj#L61-L65), you can get the data to log using `(:vars opts)`. 

The `vars-for-types` function is for getting all variables which only use values of the given `types`. It may not always get called exactly once per run at the start of the run.

Reply:
Does `(:vars opts)` contain only variables which use values of the given `types`? If so, good, cause that's why we were printing it in `vars-for-types`